src/commands/Ping.ts

```diff
@@ -0,0 +1,31 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { CommandInteraction } from 'discord.js';
+import Command from '../Command';
+import { BotClient } from '../types';
+
+/**
+ * Pings the user.
+ * Copied verbatim from BreadBot.
+ *
+ * Test Command left from the boilerplate.
```

Comment:
Good choice to leave the Ping command in!

This is safe to have in case of the bot going down, so that we know whether it's actually down or if there's something else that's wrong with the bot. This is the closest we can get to a service health check in a Discord bot.